build.gradle

```diff
@@ -49,7 +50,7 @@ configurations {
 }
 
 dependencies {
-	enigmaRuntime "net.fabricmc:stitch:0.2.1.61"
+	enigmaRuntime "net.fabricmc:stitch:0.3.0.65"
```

Comment:
Why not 0.3.1 for both stitch?

Reply:
0.3.1 doesn't exist, I need FabricMC/stitch#20 to get merged and then I will change both.